dask/dataframe/io/orc/core.py

```diff
@@ -219,13 +221,12 @@ def to_orc(
                 filename,
             ],
         )
```

Comment:
This reads to me as equivalent to:
```python
part_tasks = dsk.keys()
```

Reply:
Good point, we can avoid appending to a list here